### ISSUE TITLE
New Youtube video for IIIF viewer needs to be created 

### ISSUE THREAD

**Issue by laurensorensen** [sul-dlss/sul-embed#2135]
### Expected behavior
Youtube video follows some basic accessibility. Here is the feedback from our website accessibility team lead on Website team: 
https://youtu.be/uih5JuQnYuo?feature=shared

"The video doesn't have audio. This is a bit problematic since it is presenting information in only one format (visual). It would be better to have the steps narrated with closed captions. At a minimum, there should be a closed caption comment, in brackets [], explaining the what the video is about, that there is no audio, and where the information presented can be accessed in another format. See WCAG 1.2.1 https://www.w3.org/WAI/WCAG21/Understanding/audio-only-and-video-only-prerecorded.html." 

### Actual behavior
Video has no audio (thus no captions), should have a voiceover
Should also have a clicker that is visible/highlights the clicks and important areas in video. 

### Steps to reproduce the behavior
Look at the video, recreate 

### Browser / Environment
All browsers 

Please alert @laurensorensen when it's done, so we can update it on the website. Related JIRA ticket: https://jirasul.stanford.edu/jira/browse/WEB5-64

**Comment by andrewjbtw** [comment]
This is on hold waiting for Mirador 4.
